be/src/exec/analytic_eval_node.cpp

```diff
@@ -195,9 +194,19 @@ Status AnalyticEvalNode::open(RuntimeState* state) {
     RETURN_IF_CANCELLED(state);
     //RETURN_IF_ERROR(QueryMaintenance(state));
     RETURN_IF_ERROR(child(0)->open(state));
-    // RETURN_IF_ERROR(state->block_mgr()->RegisterClient(2, mem_tracker(), state, &client_));
-    _input_stream.reset(new BufferedTupleStream(state, child(0)->row_desc(), state->block_mgr()));
-    RETURN_IF_ERROR(_input_stream->init(runtime_profile()));
+    RETURN_IF_ERROR(state->block_mgr2()->register_client(2, mem_tracker(), state, &client_));
+    _input_stream.reset(new BufferedTupleStream2(state, child(0)->row_desc(), state->block_mgr2(), client_, false, true));
+    RETURN_IF_ERROR(_input_stream->init(id(), runtime_profile(), true));
+
+    bool got_read_buffer;
```

Comment:
would better give a better name.

Reply:
@kangkaisen Hello, I could not think of a better variable name. Could you please give me a suggestion. Thank you.

Reply:
is_got_buffer or has_got_buffer?

Reply:
ok